packages/page-settings/src/Developer.tsx

```diff
@@ -15,7 +15,7 @@ import { isJsonObject, stringToU8a, u8aToString } from '@polkadot/util';
 
 import { useTranslation } from './translate';
 
-const EMPTY_CODE = `{}`;
+const EMPTY_CODE = '{}';
```

Comment:
Do not include changes to PR that are not related to its main purpose

Reply:
This change is also from eslint